### DISCUSSION TITLE
Loadin superset in iframe different domain

### DISCUSSION THREAD

**Discussion by ajknol** [apache/superset#16122] [Q&A / Help] [answered]
### Expected results

Superset dashboard to load in iframe on website different domain

### Actual results

Unexpected error message 'Unexpected token < in JSON at position 0' shown on charts

![screen](https://user-images.githubusercontent.com/9432283/128508577-b60c751f-fe28-4dce-8c7d-46164ec96110.png)

#### How to reproduce the bug

<!DOCTYPE html>
<html>
  <head>
    <title>title</title>

  </head>
  <body>
	<iframe src="https://superset-dashboard-url" height="900" width="1900" title="Iframe Example"></iframe>
  </body>
</html>

### Environment

- superset version: `0.0.0dev`
- python version: `python 3.8.10`

**Comment by cdmikechen** [comment]
maybe try to use firefox or Chrome under 80?

**Reply by ajknol** [reply]
Thanks for your advice, but a lower Chrome version is not workable for users I am afraid (dashboard should work on new browser versions too). 
Without iframe the dashboard is working fine, within the iframe it is not unfortunately. It seems to be something with CORS to allow the dashboard to be loaded from another domain origin.

**Comment by shubhk97** [comment, marked as answer]
Use these settings in superset_config.py

SESSION_COOKIE_HTTPONLY = False 
SESSION_COOKIE_SECURE = True 
SESSION_COOKIE_SAMESITE = "None"
WTF_CSRF_ENABLED = False

**Reply by ajknol** [reply]
Thanks, works!
